### PR TITLE
Rename camelCase in windows folder

### PR DESCRIPTION
**Pull request by petergaal**

### CODE REVIEW

**Comment by duncanspumpkin** [comment]
Argh sorry should have made myself clearer. camelCase for functions and (variables but leave that to a later refactor). TitleCase for namespaces, types and files (but I'd suggest just focussing on the functions for right now).

**Comment by janisozaur** [comment]
@petergaal please mind the subject lines of commits.

```
Short (less than 72, ideally shorter) subject

Followed by single blank line
With more description in rest of the body
```

File: src/openloco/windows/BuildVehicle.cpp [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 using namespace openloco::interop;
 
-namespace openloco::ui::build_vehicle
+namespace openloco::ui::buildVehicle
```
Comment (duncanspumpkin) [inline src/openloco/windows/BuildVehicle.cpp:24]: Could you change this to TitleCase

**Review by duncanspumpkin — APPROVED** [review]
Change the buildVehicle namespace to title case and this is good to go.

File: src/openloco/windows/BuildVehicle.cpp (hunk 1 above)
Comment (petergaal) [reply src/openloco/windows/BuildVehicle.cpp:24]: I will do those changes as well later.

**Pull request merged**